.github/workflows/build.yaml

```diff
@@ -60,6 +60,17 @@ jobs:
     name: Run tests
     runs-on: ubuntu-latest
     steps:
+      - name: Maximize build space
+        uses: AdityaGarg8/remove-unwanted-software@v5
+        with:
+          remove-android: 'true'
+          remove-dotnet: 'true'
+          remove-haskell: 'true'
+          remove-codeql: 'true'
+          remove-docker-images: 'true'
+          remove-large-packages: 'true'
+          remove-cached-tools: 'true'
+          remove-swapfile: 'true'
```

Comment:
note: remove this prior to merge as it's only for testing purposes.

Reply:
IIUC it may not help us, 
can we use a large runner for releasing?